liblwgeom/lwrandom.c

```diff
@@ -0,0 +1,278 @@
+/**********************************************************************
+ *
+ * PostGIS - Spatial Types for PostgreSQL
+ * http://postgis.net
+ *
+ * PostGIS is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * PostGIS is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with PostGIS.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ **********************************************************************
+ *
+ * Copyright 2019 Mike Taves
+ *
+ **********************************************************************/
+
+
+#include <stdlib.h>
+#include <stdint.h>
+/* #include <stdio.h> */
```

Comment:
Can you remove the extra comments? Both https://github.com/postgis/postgis/pull/365/files#diff-ad20971b4155780e40b871996e30aaa6R28  and https://github.com/postgis/postgis/pull/365/files#diff-ad20971b4155780e40b871996e30aaa6R28https://github.com/postgis/postgis/pull/365/files#diff-ad20971b4155780e40b871996e30aaa6R28